api/dataset.go

```diff
@@ -94,8 +94,56 @@ func (api *API) getDatasetJSON(ctx context.Context, r *http.Request, params *dat
 	return response, nil
 }
 
+func (api *API) validateGeography(ctx context.Context, r *http.Request, params *datasetParams, datasetRequest cantabular.StaticDatasetQueryRequest) ([]string, *[]cantabular.Filter, error) {
```

Comment:
Never any need to make a pointer to a slice as a slice essentially acts like a pointer in itself (including being able to equal nil)